integration/tests/api/docker_test.go

```diff
@@ -295,6 +295,12 @@ func TestDockerFilesystemStats(t *testing.T) {
 	fm := framework.New(t)
 	defer fm.Cleanup()
 
+	storageDriver := fm.Docker().StorageDriver()
+	if storageDriver == framework.DeviceMapper {
```

Comment:
Logs usage is still supported with devicemapper. If you look below, only the root filesystem usage part is skipped for devicemapper.